news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,87 @@
+
+This time around, the hints were "*Stained in the hues of madness, they rage within a four-walled jail.*" and "*Bask in the shade of an ailing mind.*"
+
+With the fires of discovery lit in the hearts of medal aficionados across the globe, the [Medal Hunters Discord server](https://discord.gg/8qpNTs6) huddled together and decided to pool their efforts to see one of their number made the latest "**Medal Hunter**".
+
+But before we get to community's attempts to solve it, here's some details they knew from the beginning from their familiarity with previous releases:
+
+- The medal is only visible on the profiles of osu! players, so it can only be obtained by clicking circles; 
+- If the medal is thought to be specific to a particular single map, it is *usually* a Featured Artist song; 
+- The medal is in the Hush-Hush (Expert) category, which means no [difficulty reduction mods](/wiki/Gameplay/Game_modifier#difficulty-reduction) (or [special mods](/wiki/Gameplay/Game_modifier#special)) are allowed.
+
+Within the first few minutes of the medal releasing, players immediately began poring over all this information and quickly noticed the medal icon itself clearly referenced a popular Frums track titled *theyaremanycolors*. From there, aspiring medal hunters were swift to piece together that the hues from medal hint/flair may be hinting at the RGB color spectrum in some capacity, but nobody could quite figure out how. Fixated on this particular fact, much of the early exploration of the medal neglected other parts of the hint that would later prove to be integral in deciphering the riddle once and for all.
+
+Setting out to find how to translate these RGB-related suspicions into in-game scores, much was considered and tried. Red, green, and blue imagery all shows up prominently in the video associated with many *theyaremanycolors* beatmaps, so many initial theories involved exploring failing at particular parts of the video, or only comboing up to the first appearances of specific colours. Speculations grew fast and wild - could it have been to do with setting certain grades of scores back-to-back? After all, as every (mostly) sane prospective medal hunter knows,, D ranks are red, A ranks are green, and B ranks are blue.
+
+When this did not bear any fruit, considerations started turning towards mod colours In the default skin, Hard Rock is red, Easy is green-- but wait... Expert Hush-Hush means absolutely no difficulty reducing mods, so that couldn't be it. And while "*hues of madness*" seemed to be critical in figuring out the medal solution, just what in the world could "*Bask in the shade of an ailing mind*" mean? 
```

Comment:
```suggestion
When this did not bear any fruit, considerations started turning towards mod colours in the default skin, Hard Rock is red, Easy is green — but wait... Expert Hush-Hush means absolutely no difficulty reducing mods, so that couldn't be it. And while "*hues of madness*" seemed to be critical in figuring out the medal solution, just what in the world could "*Bask in the shade of an ailing mind*" mean? 
```